src/components/NoProjectionFound.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Message } from 'semantic-ui-react';
+
+const requiredFieldsForProjection = [
+  'forestEcoregion',
+  'forestType',
+  'altitudinalZone',
+];
+
+function NoProjectionFound() {
+  const { mapLocation, projectionMode, projectionLocation } = useSelector(
+    state => ({
+      mapLocation: state.mapLocation,
+      projectionMode: state.projectionMode,
+      projectionLocation: state.projectionLocation,
+    }),
+  );
+
+  const { t } = useTranslation();
+  const missingFields = requiredFieldsForProjection
+    .filter(field => mapLocation[field] === undefined)
+    .map(field => t(`${field}.label`));
+
+  return (
+    <>
+      {projectionMode === 'map' &&
```

Comment:
We can wrap the whole thing in "projectionMode === 'map'" and avoid the React fragment.

Reply:
it throws an error if React fragment is not provided.

Reply:
Wrap the whole return value in "projectionMode === 'map'", that should work.

Reply:
removed the second error message which was just a fallback. Now with only one error message remaining, wrapped the whole return value.